docs/ref/forms/fields.txt

```diff
@@ -1215,6 +1215,9 @@ method::
         field will be derived, and which will be used to validate the
         user's selection.
 
+    ``ModelMultipleChoiceField`` also takes one optional argument,
```

Comment:
What about `empty_label`? I think that also applies to this subclass. If we don't repeat the description for the optional arguments, I don't think we need to repeat the description for the required `queryset` argument either, so maybe you could restructure that.